src/app/pages/deprecation/DeprecationAction.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace app\pages\deprecation;
+
+use Slim\Psr7\Request;
+use Slim\Views\Twig;
+use app\domain\Version;
+use app\domain\util\Redirect;
+use app\domain\Artifact;
+
+class DeprecationAction
+{
+  private Twig $view;
+
+  public function __construct(Twig $view)
+  {
+    $this->view = $view;
+  }
+
+  public function __invoke(Request $request, $response, $args)
+  {
+    return $this->view->render($response, 'deprecation/deprecation.twig', [
+      'versions' => self::getVersions(),
+      'features' => self::getFeatures()
+    ]);
+  }
+  
+  private static function getVersions(): array
+  {
+    return ["3", "4", "5", "6", "7", "8", "9.1", "9.2", "9.3"];
+  }
+  
+  private static function getFeatures(): array
+  {
+    return [
+      new Feature(
+        "CMS Page", 
+        "/doc/8.0/designer-guide/user-interface/web-page/html-content-in-the-cms.html",
+        "Html Dialog", 
+        "/doc/latest/designer-guide/user-interface/user-dialogs/html-dialogs.html",
```

Comment:
we do no longer reference to "latest" version. because "latest" points to latest LTS version. as soon as we release next LTS version all links can be broken. Just use a stable LTS link e.g. 8.0.